apps/application-system/api/src/app/modules/payment/payment.service.ts

```diff
@@ -12,7 +12,7 @@ import type { User } from '@island.is/auth-nest-tools'
 import { CreateChargeResult } from './payment.type'
 import { logger } from '@island.is/logging'
 import { ApolloError } from 'apollo-server-express'
-import { Application as ApplicationModel } from '../application/application.model'
+import { Application as ApplicationWithAttachments } from '../application/application.model'
```

Comment:
Why import the `Application` model without attachments to rename it to `ApplicationWithAttachments` ?

Reply:
yes, looks like it.. I'll fix that